apps/evm/src/clients/api/queries/getUserVaiBorrowBalance/index.ts

```diff
@@ -0,0 +1,50 @@
+import BigNumber from 'bignumber.js';
+import { vaiControllerAbi } from 'libs/contracts';
+import { VError } from 'libs/errors';
+import type { Address, PublicClient } from 'viem';
+
+export interface GetUserVaiBorrowBalanceInput {
+  accountAddress: string;
+  vaiControllerContractAddress: Address;
```

Comment:
Similar to the other comments, I'm in doubt about when `accountAddress`should be `Address` or `string` 🤔

Reply:
Let's use `Address` everywhere, so we're consistent with `viem`.